### PR TITLE
Updated generated doc samples with formatting functions

### PR DESCRIPTION
**Pull request by michaelbausor**

### CODE REVIEW

File: vgen/src/main/resources/io/gapi/vgen/java/main.snip [hunk 1]
```diff
@@ -626,6 +626,20 @@
...
+  @if param.hasFormattingConfig() {
+    {@ApiName}.{@formatResourceFunctionName(param.getFormattingConfig())}({@formatResourceFunctionArgs(param.getFormattingConfig())})
+  @else
+    {@context.zeroValue(param.getType)}
+  @end
+@end
+
+@private formatResourceFunctionArgs(collectionConfig)
+  @join param : collectionConfig.getNameTemplate.vars() on ", "
+    "[{@context.lowerUnderscoreToLowerCamel(param).toUpperCase()}]"
```
Comment (garrettjonesgoogle) [inline vgen/src/main/resources/io/gapi/vgen/java/main.snip]: maybe use lowerUnderscoreToUpperUnderscore

File: vgen/src/main/java/io/gapi/vgen/java/JavaContextCommon.java [hunk 2]
```diff
@@ -218,7 +236,9 @@ public Builder setParams(
                 s_newVariable(
                     field.getType(),
                     "formatted" + LanguageUtil.lowerUnderscoreToUpperCamel(field.getSimpleName()),
-                    languageProvider.getDescription(field)));
+                    languageProvider.getDescription(field),
+                    languageProvider.getCollectionConfig(
+                        service, fieldNamePatterns.get(field.getSimpleName()))));
```
Comment (garrettjonesgoogle) [inline vgen/src/main/java/io/gapi/vgen/java/JavaContextCommon.java]: why does getCollectionConfig need the service?

Comment (michaelbausor) [inline vgen/src/main/java/io/gapi/vgen/java/JavaContextCommon.java]: To retrieve the InterfaceConfig - see here:
https://github.com/googleapis/toolkit/blob/master/vgen/src/main/java/io/gapi/vgen/java/JavaGapicContext.java#L341

The languageProvider.getCollectionConfig(service, name) method is calling languageProvider.getApiConfig().getInterfaceConfig(service).getCollectionConfig(name)

File: vgen/src/main/resources/io/gapi/vgen/java/main.snip (hunk 1 above)
Comment (michaelbausor) [inline vgen/src/main/resources/io/gapi/vgen/java/main.snip]: Fixed

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**